src/navigator-card-body.jsx

```diff
@@ -374,6 +301,7 @@ export const NavigatorCardBody = ({
 const Item = React.memo(function Item({ file, isSelected, setSelected, onDoubleClickNavigate, isGrid }) {
     function handleClick(ev, file) {
         if (ev.detail > 1) {
+            setSelected([]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-307-20240307-112122-d90b1ce5-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)